app/common/data/interfaces/temporary.py

```diff
@@ -100,9 +100,9 @@ def delete_question(question: Question) -> None:
     # todo: when/if this becomes a non-temporary interface, TEST THOROUGHLY. The OrderingList we're using for this
     # definitely has a few quirks.
     db.session.delete(question)
-    if question in question.form.questions:
-        question.form.components.remove(question)
-    question.form.components.reorder()
+    if question in question.contained_by.components:
```

Comment:
_Might_ want to call it `container` instead of `contained_by`, which makes me think of created_by/updated_by which are users. But that feels very personal preference so - no big feeling